heat_infoblox/object_manipulator.py

```
@@ -170,17 +192,58 @@ def update_ns_group(self, group_name, group):
         self._update_infoblox_object('nsgroup', {'name': group_name},
                                      group)
 
-    def create_ospf(self, member_name, ospf_options_dict):
-        """Add ospf settings to the grid member."""
+    @staticmethod
+    def _copy_fields_or_raise(source_dict, dest_dict, fields):
+        for field in fields:
+            if field not in source_dict:
+                raise ValueError(_("Field '{}' is required").format(field))
```

This will not bubble up to heat if you need to bubble up the exception.